account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -56,8 +75,7 @@ def extend_payment_order_domain(self, payment_order, domain):
             # Do not propose partially reconciled credit lines,
             # as they are deducted from a customer invoice, and
             # will not be refunded with a payment.
-            domain += [('credit', '>', 0),
-                       '|',
+            domain += ['|',
```

Comment:
Is that consistent with the fact that further in the code we select the residual amount? At the very least, the comment just above is not correct anymore.